apps/server/src/utils/ElasticSearchClient.ts

```diff
@@ -0,0 +1,100 @@
+import type { estypes } from '@elastic/elasticsearch'
+import { Client } from '@elastic/elasticsearch'
+import Mock from '@elastic/elasticsearch-mock'
+import { readFileSync } from 'fs'
+import { resolve } from 'path'
+import config from '../config'
+
+import type { CachedInstitution } from 'src/shared/contract'
+
+function getInstitutionFilePath () {
+  if (config.Env === 'test') {
+    console.log('loading test institutions')
+    return resolve(__dirname, '../../cachedDefaults/testInstitutionsMapping.json')
+  } else {
+    console.log('loading all institutions into elasticSearch')
+    return resolve(__dirname, '../../cachedDefaults/ucwInstitutionsMapping.json')
+  }
+}
+
+export const ElasticSearchMock = new Mock()
+
+export const ElasticsearchClient = new Client({
+  node: config.ELASTIC_SEARCH_URL ?? 'http://localhost:9200',
+  ...(process.env.NODE_ENV === 'test' && { Connection: ElasticSearchMock.getConnection() })
+})
+
+export async function initialize (client: Client) {
```

Comment:
Do we need to pass this client around, or can these functions just call ElasticsearchClient directly

Reply:
I don't think so I was planning on refactoring that this morning